Algorithm.Framework/QuantConnect.Algorithm.Framework.csproj

```diff
@@ -123,6 +140,10 @@
       <Project>{73fb2522-c3ed-4e47-8e3d-afad48a6b888}</Project>
       <Name>QuantConnect.Indicators</Name>
     </ProjectReference>
+    <ProjectReference Include="..\packages\alglib2.dll.3.10.4\Alglib2.csproj">
```

Comment:
Interesting... this nuget package comes with the entire source code? Does this **have** to be a project reference?